frontend/src/metabase/components/ViewFooterButton/ViewFooterButton.tsx

```diff
@@ -17,12 +17,12 @@ export type ViewFooterButtonProps = {
 
 export const ViewFooterButton = forwardRef(function _ViewFooterButton(
   { icon, tooltipLabel, ...actionIconProps }: ViewFooterButtonProps,
-  ref: Ref<HTMLButtonElement>,
+  ref: Ref<HTMLDivElement>,
```

Comment:
We were getting an error in the browser related to `aria-disabled` and how we couldn't set the button to `disabled=true`. This seems to be coming from setting the `ref` to the `ActionIcon` (which is a `button`)